buildtest/schemas/spack.schema.json

```diff
@@ -118,6 +118,11 @@
             }
           }
         },
+        "deactivate": {
+          "additionalProperties": false,
```

Comment:
there is no need for `additionalProperties` this is only needed if you have `type: object` which in YAML is used to define multiple properties.

Reply:
I have fixed it.
Thank you.